lib/filecoin-rpc-client.js

```diff
@@ -0,0 +1,41 @@
+import pRetry from 'p-retry'
+/**
+ * @param {object} options
+ * @param {number} [options.maxAttempts]
+ * @param {function} [options.rpcFn] The RPC function to use
```

Comment:
```suggestion
 * @param {function} options.rpcFn The RPC function to use
```

This parameter isn't optional